pkg/render/guardian.go

```diff
@@ -708,5 +717,23 @@ func deprecatedObjects() []client.Object {
 			TypeMeta:   metav1.TypeMeta{Kind: "ClusterRoleBinding", APIVersion: "rbac.authorization.k8s.io/v1"},
 			ObjectMeta: metav1.ObjectMeta{Name: "tigera-guardian"},
 		},
+
+		// Remove manager namespace objects since the guardian identity is responsible for handling manager requests
```

Comment:
Do we need to do the same for tigera-elasticsearch, tigera-intrusion-detection, etc.?

Reply:
Ah, I see some of them are handled in other controllers. Is there a reason to remove the manager resources from the guardian controller, but the other resources from their own controllers?

Reply:
Because in managed clusters, Guardian was responsible for creating the manager namespace and its associated ClusterRoles (as shown above in line 154).